projects/gateway/pkg/validation/validator.go

```diff
@@ -72,7 +72,7 @@ var (
 	}
 
 	mValidConfig = utils2.MakeGauge("validation.gateway.solo.io/valid_config",
-		"A boolean indicating whether gloo config is valid")
+		"A boolean meant to indicate whether the gloo config is valid. However, its behavior/meaning changes depending upon the validation configuration and in most cases there is a better metric to use")
```

Comment:
Is this user facing? 
if not feel free to ignore/i'll resolve
if it is, could we potentially point people to the better metrics?
```suggestion
		"A boolean that indicates whether the Gloo configuration is valid. However, its behavior changes depending upon the validation configuration, and in most cases there is a better metric to use.")
```

Reply:
That auto-closed because I accepted the commit... I wanted to get the phrasing changes in. I will add the link to relevant docs, the other file being modified in this PR. ( https://docs.solo.io/gloo-edge/latest/guides/traffic_management/configuration_validation/ )
)